### PR TITLE
chore(benchmarks): run benchmarks in firefox and fix docker image

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ Uses the image configuration from Playwright https://github.com/microsoft/playwright/blob/master/docs/docker/Dockerfile.bionic
+ Added some extra package/steps to make it work for both our karma and playwright benchmarks.

### CODE REVIEW

File: .ci/docker/node-playwright/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y \
+        google-chrome-unstable \
+        fonts-ipafont-gothic \
+        fonts-wqy-zenhei \
+        fonts-thai-tlwg \
+        fonts-kacst \
+        --no-install-recommends \
+        && rm -rf /var/lib/apt/lists/*
```
Comment (v1v) [inline .ci/docker/node-playwright/Dockerfile]:
```suggestion
RUN apt-get install -y wget --no-install-recommends \
    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
    && apt-get update \
    && apt-get install -y \
        google-chrome-unstable \
        fonts-ipafont-gothic \
        fonts-wqy-zenhei \
        fonts-thai-tlwg \
        fonts-kacst \
        --no-install-recommends
```

File: .ci/docker/node-playwright/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# 6. Install Firefox dependencies
+RUN apt-get update \
+    && apt-get install -y libdbus-glib-1-2 \
+                          xvfb
```
Comment (v1v) [inline .ci/docker/node-playwright/Dockerfile]:
```suggestion
RUN apt-get install -y libdbus-glib-1-2 \
                          xvfb

```

Comment (vigneshshanmugam) [reply .ci/docker/node-playwright/Dockerfile]: I was getting these errors if i dont update 

```
14:09:44  Service 'node-benchmark' failed to build: The command '/bin/sh -c apt-get install -y libdbus-glib-1-2                           xvfb' returned a non-zero code: 100
```

Comment (v1v) [reply .ci/docker/node-playwright/Dockerfile]: Sure thing that was caused by https://github.com/elastic/apm-agent-rum-js/pull/683/files#diff-421113117290176fb1a618548b8c9943R48

**Comment by vigneshshanmugam** [comment]
The nit suggestions are applied on the next PR #685

**Pull request merged**